to.us.scrollprotocol.Profectus.yml

```diff
@@ -0,0 +1,66 @@
+id: to.us.scrollprotocol.Profectus
```

Comment:
`scrollprotocol.to.us` is NXDOMAIN (domain doesn't exist). Yet you acknowledged:

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.